util/create-brackets-table.raku

```diff
@@ -1,74 +1,102 @@
 #!/usr/bin/env raku
 
-# global array defined in the BEGIN block at the end:
-my @bracket-chars;
+# TODO
+#   + get desired exit behavior from Coke
+#   + settle on table name and output file name
+
+# Global array initially defined in the BEGIN block at the end.
+# The array is then defined dynamically by reading the source
+# file at https://github.com/Raku/nqp/src/HLL/Grammar.nqp.
+our @bracket-chars;
 
-my $f = "../doc/Language/brackets.pod6";
 if !@*ARGS {
     say qq:to/HERE/;
-    Usage: {$*PROGRAM.IO.basename} go | reorder
+    Usage: {$*PROGRAM.IO.basename} go | Reorder [refresh][original][debug]
 
     Writes the HLL::Grammar '\$brackets' chars into a Pod6 table.
 
-    If the 'reorder' mode is chosen, the output table is sorted
+    If the 'Reorder' mode is chosen, the output table is sorted
     by the value of the codepoint of the opening character.
     Otherwise, the order of the source definition is used.
+
+    The HLL::Grammar.nqp file as source is updated if it
+    is not found in /util or the 'refresh' option is used.
+
+    Use the 'original' option to use the internal source of
+    the brackets.
     HERE
     exit;
 }
 
-my $reorder = 0;
-my $arg = @*ARGS.shift;
-if $arg ~~ /:i r/ {
-    ++$reorder;
-}
-say "Reorder = $reorder";
-
-my @ofils;
+my $reorder       = 0;
+my $refresh       = 0;
+my $want-original = 0;
+my $debug         = 0;
 
-my $lb = '|';
-my $rb = '|';
+for @*ARGS {
+    when /^ r/ { ++$refresh }
+    when /^ R/ { ++$reorder }
+    when /^ o/ { ++$want-original }
+    when /^ d/ { ++$debug }
+}
 
+# The output table file:
+my $repopath = $*PROGRAM.IO.absolute.IO.parent(2);
+my $fdir1 = $repopath ~ "/doc/Language";
+my $f1    = "$fdir1/brackets.pod6";
```

Comment:
You have based this against an old version - I've already made similar changes to the file that are committed in the repository.

You'll have to rebase on your end and fix the PR or submit a new one.